exchanges.yml

```diff
@@ -0,0 +1,517 @@
+            - type: object
+              description: An alternate presentation schema format to use when validating a presentation in the current step. Additional properties are expected for a specific presentation schema type, but are beyond the scope of this specification.
+              properties:
+                type:
+                  type: string
+                  description: The type of presentation schema mechanism to use when validating the presentation.
+        verifyPresentationResponseSchema:
+          oneOf:
+            - type: object
+              description: A JSON Schema describing the validation to be performed on the result of verifying the presentation.
+              properties:
+                  type:
+                    type: string
+                    description: The value MUST be <code>JsonSchema</code>.
+                  jsonSchema:
+                    type: object
+                    description: The JSON Schema object to use when validating the result of verifying the presentation.
+            - type: object
+              description: An alternate presentation schema format to use on the result of verifying a presentation in the current step. Additional properties are expected for a specific presentation schema type, but are beyond the scope of this specification.
+              properties:
+                type:
+                  type: string
+                  description: The type of presentation schema mechanism to use when validating the result of verifying the presentation.
+        nextStep:
+          type: string
+          description: The name of the next step in the sequence. Passing nextStep is OPTIONAL. This field MUST NOT be present on the final step configuration.
+        openId:
+          type: object
+          description: Optional information to trigger OID4VP
+          properties:
+            createAuthorizationRequest:
+              type: string
+              description: If present, identifies the name of the template variable (in <code>variables</code> for an <a href="#create-exchange">exchange</a>) that the auto-generated OpenID for Verifiable Presentations Authorization Request will be stored in for subsequent use in the exchange. Either this or <code>authorizationRequest</code> MUST be present, but not both. The value of the named variable can be lazily generated when the Authorization Request is required, i.e., after the client of the exchange chooses to use the OID4VP protocol. This feature allows the Authorization Request to be generated by reusing values, e.g., a Verifiable Presentation Request, that only need to be specified once across multiple protocol offerings.
+            authorizationRequest:
+              type: object
+              description: If present, an OID4VP Authorization Request for subsequent use in the exchange. Either this or <code>createAuthorizationRequest</code> MUST be present, but not both.
```

Comment:
Why sometimes "OID4VP" and sometimes "OpenID for Verifiable Presentations"? Better to be consistent, which I would suggest be either "OID4VP (OpenID for Verifiable Presentations)" or just "OID4VP" if there's a guaranteed easily-accessible place that the reader can see the expansion.